packages/frontend-2/components/workspace/ProjectList.vue

```diff
@@ -32,9 +32,30 @@
             v-bind="bind"
             v-on="on"
           />
-          <FormButton v-if="!isWorkspaceGuest" @click="openNewProject = true">
-            New project
-          </FormButton>
+          <div class="flex gap-2">
+            <!--- Conditionally apply tooltip only for non-admins and avoid v-tippy reactivity bug -->
+            <div v-if="!isWorkspaceAdmin" v-tippy="'You must be a workspace admin'">
```

Comment:
@benjaminvo This is different tooltip text to what you suggested in Linear. It seems our tooltips cant handle longer text right now. 

I've shortened it, but feel free to update this, or suggest something better. I can also look at improving the tooltips, and going with the original text if you think it is needed. 

![image](https://github.com/user-attachments/assets/6ec4b5d9-1e75-4dbc-adc4-de868f357bf2)

Reply:
I like it!